buildSrc/src/main/kotlin/strings.kt

```diff
@@ -0,0 +1,10 @@
+/**
+ * Convert pascalCase to SCREAMING_SNAKE_CASE
+ */
+fun String.toScreamingSnakeCase() = map {
```

Comment:
huh... alright.

Reply:
lol. Oh I forgot to add the tests for this. In fact do you want test for all the gradle stuff? Most of it's pretty easy to know if it's working correctly or not though.

Reply:
Nah, it's not like it's an api